Makefile

```diff
@@ -36,6 +39,8 @@ docker/pre-commit-hooks:
 	@echo "${GREEN}Start running the pre-commit hooks with docker${RESET}"
 	@docker run --rm \
 		-v ${PWD}:${MOUNT_TARGET_DIRECTORY} \
+		-u ${USER_UID}:${USER_GID} \
+		-e HOME=/tmp \
```

Comment:
why is it mandatory to change the `HOME` path in this case ?

Reply:
Because else Home is / and current user can not create filles there

Reply:
But we don't need to create any files after all?  Also, we should set the home inside the Dockerfile already then instead of duplicating it across all Makefiles.

Reply:
Pre-commit creates filles in user home .cache

User home is determined by passwd file and set to / when user not in passed .. so no way that I know to set it inside docker without creating a fancy entrypoint wrapper  to do so..  might be an option.. but this ends up doing yak shaving if you g go down that path for everything.. ;) should put it on the list though